react/AsidedLayout/AsidedLayout.js

```diff
@@ -0,0 +1,30 @@
+import styles from './AsidedLayout.less';
+import React from 'react';
+import classnames from 'classnames';
+
+const defaultRenderAside = () => null;
+
+export default function AsidedLayout({ className, children, renderAside = defaultRenderAside, size, reverse }) {
+  return (
+    <div
+      className={classnames({
+        [className]: className,
+        [styles.root]: true,
+        [styles.reverse]: reverse
+      })}>
+      { reverse ?
```

Comment:
The use case is when the sidebar contains important contextual information, but is much, much smaller than the main content. If you're reading the content top-to-bottom, it's important to read it first, but it can happily sit on the right-hand side if your screen is large enough. Make sense?